README.md

```diff
@@ -122,28 +129,28 @@ The interaction with OpenShift is via the command line tool _oc_ which is copied
            To login as administrator:
                oc login -u system:admin
 
-    Note that, the IP seen above is dynamic and can change. It can be retrieved with `minishift ip`. Also,
-    instead of 'kvm', you will see 'xhyve' on Mac OS and 'hyperv' on Windows.
+  **Note**:
 
-    **Note:** By default Minishift uses the driver most relevant to the host OS.
-To use a driver of choice for Minishift, use the `--vm-driver=xxx` flag with `minishift start`. For example, to use VirtualBox instead of KVM for Fedora, use `minishift start --vm-driver=virtualbox`.
+  - The IP is dynamically generated for each OpenShift cluster. To check the IP, run the `minishift ip` command.
+  - By default Minishift uses the driver most relevant to the host OS. To use a different driver, set the `--vm-driver` flag in `minishift start`. For example, to use VirtualBox instead of KVM on Linux operating systems, run `minishift start --vm-driver=virtualbox`. For more information about `minishift start` options,
+  see the [minishift start command reference](/docs/minishift_start.md).
 
-1. Add `oc` binary to the _PATH_:
-
-     **Note:** How to modify the _PATH_ varies depending on host OS, version of the OC binary and
-     other variables. In case of doubt, you can check the content of the
-     `~/.minishift/cache/oc` directory.
+1. Add the `oc` binary to the _PATH_ environment variable.
 
         $ export PATH=$PATH:~/.minishift/cache/oc/v1.3.1
 
-1. Login to your OpenShift account and authenticate yourself:
-
-        $ oc login https://192.168.99.128:8443 -u developer -p developer
+  **Note:** Depending on the operating system and the `oc` version, you might need
```

Comment:
> **You are logged in as:
> User: developer
> Password: developer

Funny, which version of minishift are you using. I don't even see this. But that's besides the point really.

> In that case the last 5 lines of the output for minishift start is misleading, 

No, it might be that you are logged in, but my point is that this is a side effect of being logged in before. If you for example would do a _oc logout_ you will also see that login is required.

> At the first line of the output itself we speak about starting an OC instance, so it is natural for user to assume that this for OC.

Now, that's not true. Nowhere in the output it says that "OC" would be started. We are starting/provisioning OpenShift.

> We can always add the step back if it's a required one, 

+1

> which leaves us with the confusing output message.

Not at all, all is consistent. Also, in this tutorial we assume a clean install. Maybe for completeness sake we can add _oc logout_ somewhere at the end of the creating app example. But to be honest, no one does that. You want _oc_ to stay logged in, otherwise you would have to provide credentials for each and every single command you are running.